agent/cmd/cuda-checkpoint-helper/transfer_engine_test.cpp

```
@@ -0,0 +1,26 @@
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2026 NVIDIA CORPORATION & AFFILIATES.
+ * All rights reserved. SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "transfer_engine.h"
+
+#include <cassert>
+#include <chrono>
+
+int main() {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)